docs/cql/ddl.rst

```diff
@@ -975,6 +975,9 @@ The ``ALTER TABLE`` statement can:
    Scylla allows the client to provide any timestamp on any table, it is theoretically possible to use another
    convention. Please be aware that if you do so, dropping a column will not work correctly.
 
+.. warning:: Dropping a column with timestamp specified (``ALTER TABLE ... DROP ... USING TIMESTAMP ...``) 
+   should only be used to restore schema from description (``DESCRIBE SCHEMA WITH INTERNALS``).
+
```

Comment:
Please add some words (before the warning) that explain what the timestamp means.